integration/setup/common.go

```diff
@@ -132,7 +132,7 @@ func common(ctx context.Context, config Config) error {
 					},
 					{
 						Name:    "containerlinux",
```

Comment:
is flatcar still considered containerlinux?

Reply:
AFAIK yes - it's just a fork/continuation.